docs/en_US/Release.rst

```diff
@@ -5,6 +5,69 @@
 Change Log
 ==========
 
+Release 2.1 - 3/8/2021
+-----------------------
+
+Major updates
+^^^^^^^^^^^^^
+
+Neural architecture search
+""""""""""""""""""""""""""
+
+* Improve NAS 2.0 (Retiarii) Framework (Alpha release)
+
+  * `Feature roadmap <https://github.com/microsoft/nni/issues/3301>`__
+  * `Related issues and pull requests <https://github.com/microsoft/nni/issues?q=label%3Aretiarii-v2.1>`__
+  * Documentation improvement (#3404 #3432)
+
+Training service
+""""""""""""""""
+
+* Support shared storage for reuse mode (#3354)
+* Support local platform for Windows in hybrid training service (#3353)
+* Remove PAIYarn training service (#3327)
+* Add recently-idle environment scheduler in reuse mode (#3375)
```

Comment:
-> Add "recently-idle" scheduling algorithm and set it as default scheduling algorithm in reuse mode